commands/resource_commands/deploy

```diff
@@ -0,0 +1,247 @@
+#!/bin/bash
+set -e
+. "$GRPL_WORKDIR/utils/help_menus"
+. "$GRPL_WORKDIR/utils/checks"
+
+cli_help() {
+  grpl_resource_deploy_cli_help
+  exit 0
+}
+
+[ "$1" = "help" ] || [ "$1" = "h" ] && cli_help
+
+install_prerequisite
+
+cli_log "resource deploy BEGIN"
+status_log $TYPE_DEFAULT "resource deploy is in progress"
```

Comment:
this message is ambigous - at this stage, no resource is being deployed, I guess